app/shop.py

```diff
@@ -0,0 +1,15 @@
+class Shop:
+    def __init__(self, name: str, location: list[int], products: dict) -> None:
+        self.name = name
+        self.location = location
+        self.products = products
+
+    def prise_products(
```

Comment:
The method name `prise_products` seems to be a typo. It should be `price_products` to better reflect its purpose of calculating the total price of the specified products.